src/index.html

```diff
@@ -12,24 +12,18 @@ <h1 class="main-title"> Leaderboard</h1>
          <div class="recent-scores flex-column width-50">
              <div class="scores-title flex-row">
                 <h3 class="score-title">Recent Scores</h3>
-                <button class="btn refresh-btn" type="button">Refresh</button>
-             </div>
-             <div class="scores-content">
-                 <p class="score">Name: 100</p>
-                 <p class="score">Name: 20</p>
-                 <p class="score">Name: 50</p>
-                 <p class="score">Name: 78</p>
-                 <p class="score">Name: 125</p>
-                 <p class="score">Name: 77</p>
-                 <p class="score">Name: 42</p>
+                <button id="refresh" class="btn refresh-btn" type="button">Refresh</button>
              </div>
+             <ul class="scores-content">
+            
+            </ul>
          </div>
          <form action="#" class="score-form width-50 flex-column">
              <h3 class="scores-title">Add Your Score</h3>
              <div class="container flex-column">
```

Comment:
- [ ] [Optional] adding a long score can break the view of your app, kindly try to add some kind of form validation to avoid this

![Screenshot_20220512_052141](https://user-images.githubusercontent.com/25227022/167991856-527e375f-ef1d-43c8-8ce4-946c9d7a1b0b.png)

Reply:
Thanks for noticing that. I have added validation feature for that input now 👍🏽